### ISSUE TITLE
Later images not working?

### ISSUE THREAD

**Issue by richardpenman** [originx/android_emulator_docker#1]
Thanks for sharing this.
I tried a few images and got older ones working, such as _system-images;android-18;google_apis;armeabi-v7a_ and _system-images;android-19;google_apis;armeabi-v7a_, however later ones do not load. Have you had any luck with more recent images?

**Comment by originx** [comment]
Yep I have it in plan to enable them, the latter ones use x86 arch which means the kvm, haxm or similar virtualization needs to be enabled to the docker image for example if you have a Linux host machine you start it with docker --privileged -v dev/kvm:/dev/kvm

The command is not complete ie take it as a pseudocode

**Comment by richardpenman** [comment]
Have you had any luck with later ARM images? _system-images;android-23;google_apis;armeabi-v7a_ and _system-images;android-24;google_apis;armeabi-v7a_ didn't load in my tests. _system-images;android-24;google_apis;arm64-v8a_ did load but was too slow.

**Comment by originx** [comment]
I just tested and something seems to be very broken with the emulator itself, we have seen emulator crashes on normal unix host system and reported multiple issues.
the google team is working on it but since it is a native crash I am not sure is there anything else that is possible to do I will contact google devs and ask them for assistance with a repro case
